tmpl/help.html

```diff
@@ -122,7 +122,7 @@ <h2 id="api">Application Programming Interface (API)</h2>
 <p>
 Create a new link by sending a POST request with a <code>short</code> and <code>long</code> value:
 
-<pre>{{`$ curl -L -d short=cs -d long=https://cs.github.com/ go
+<pre>{{`$ curl -L -H "Sec-Golink: 1" -d short=cs -d long=https://cs.github.com/ go
```

Comment:
This is fine as-written, but we could avoid the need for quotes by compacting the arg:

```bash
curl -L -H sec-golink:1 -d short=cs -d long=https://cs.github.com/ go
```

or similar. YMMV.

Reply:
Done